src/cudamatrix/cu-matrix.cc

```diff
@@ -1229,6 +1229,47 @@ void CuMatrixBase<Real>::AddMatMatElements(Real alpha,
   }
 }
 
+template<typename Real>
+void CuMatrixBase<Real>::ParametricReLU(const CuMatrixBase<Real> &src, const CuVectorBase<Real> &a, const CuVectorBase<Real> &b) {
```

Comment:
Please rename the method to 'ParametricRelu' (it's consistent with other methods).